packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2273,8 +2273,13 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
 
   @override
   void didChangeMetrics() {
-    if (_lastBottomViewInset < WidgetsBinding.instance!.window.viewInsets.bottom) {
-      _scheduleShowCaretOnScreen();
+    if (_lastBottomViewInset != WidgetsBinding.instance!.window.viewInsets.bottom) {
+      SchedulerBinding.instance!.addPostFrameCallback((Duration _) {
+        _selectionOverlay?.updateForScroll();
```

Comment:
[updateForScroll](https://github.com/flutter/flutter/blob/4d96a3fd508cebaf198b6da4752a51175d784cd9/packages/flutter/lib/src/widgets/text_selection.dart#L537) is not the best name for this, but it does what I want...  Any thoughts on renaming or another way to avoid the misnomer?

Reply:
Well the view does scroll up when the keyboard shows...

Reply:
Would `updateOverlay` be too general?

Reply:
There's also an `update`!  I think I'll leave it and consider this a "scroll".  Maybe we could clean this up or make this less imperative in the future, though.